include/mbgl/util/thread.hpp

```diff
@@ -127,13 +162,23 @@ class Thread : public Scheduler {
 private:
     MBGL_STORE_THREAD(tid);
 
-    void schedule(std::weak_ptr<Mailbox> mailbox) override {
-        loop->schedule(mailbox);
+    void schedule(std::weak_ptr<Mailbox> mailbox_) override {
+        assert(loop);
+        loop->schedule(mailbox_);
+    }
+    
+    template <typename ArgsTuple, std::size_t... I>
+    Actor<Object>* emplaceActor(std::shared_ptr<Mailbox> sharedMailbox, ArgsTuple args, std::index_sequence<I...>) {
+        return new (&actorStorage) Actor<Object>(std::move(sharedMailbox), std::move(std::get<I>(std::forward<ArgsTuple>(args)))...);
     }
 
+    std::weak_ptr<Mailbox> mailbox;
+    std::aligned_storage<sizeof(Actor<Object>)> actorStorage;
```

Comment:
Should be `std::aligned_storage_t<sizeof(Actor<Object>)> actorStorage;` (note `_t` suffix).

Reply:
wow, so subtle. great spot!

Reply:
🤦‍♂️  oooof.  Yeah, nice catch :)